pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -809,7 +809,7 @@ where
 
         // insert gas reservation slots
         let program_id = ProgramId::from_origin(instance.addr);
-        ProgramStorageOf::<T>::update_active_program(program_id, |program| {
+        ProgramStorageOf::<T>::update_active_program(program_id, |program, _bn_ref| {
```

Comment:
See no reason in such naming cause that we should go with program_ref as well
```suggestion
        ProgramStorageOf::<T>::update_active_program(program_id, |program, _bn| {
```

Reply:
done